uniflow/flow/flow.py

```diff
@@ -85,6 +90,8 @@ def _exit(self, nodes) -> Mapping[str, Any]:
 class LinearFlow(Flow):
     """Linear flow class."""
 
+    tag = constants.BASIC
```

Comment:
`LinearFlow` is never imported anyway after you remove it from __init__.py, so it is not registered to the factory.

You should only add tag for flows registered into the factory. `LinearFlow` is just my demo and you should not import it into the factory.

You should remove the tag.